ctapipe/io/sst1meventsource.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 from .eventsource import EventSource
 from .containers import SST1MDataContainer
-from ..instrument import TelescopeDescription
```

Comment:
did this not work?

Reply:
Sorry, I imagine it did. My previous PR was before your PR and in merging I lost that import, and pycharm's autofill chose the other import approach

Reply:
interesting ... thx.